docs/concepts/what-is-a-global-namespace.md

```diff
@@ -10,14 +10,19 @@ tags:
 A Global Namespace is a [Namespace](/concepts/what-is-a-namespace) that exists across Clusters when [Multi-Cluster Replication](/concepts/what-is-multi-cluster-replication) is set up.
 
 - [How to register a Global Namespace](/tctl/namespace/register/#--global-namespace)
+- [How to change the active Cluster for a Global Namespace](/tctl/namespace/update/#--active-cluster)
 
 The Global Namespace feature enables Workflow Executions to progress through another Cluster in the event of a failover.
 
 A Global Namespace may be replicated to any number of Clusters, but is active in only one Cluster at any given time.
 
 For a failover to be successful, Worker Processes must be polling for Tasks for the Global Namespace on all Clusters.
 
-Only the active Cluster dispatches [Tasks](/concepts/what-is-a-task).
+Global Namespaces have a failover version.
```

Comment:
Should this sentence be entirely singular instead? "A Global Namespace has a failover version."